src/main/java/spring/intro/service/impl/UserMapper.java

```diff
@@ -0,0 +1,24 @@
+package spring.intro.service.impl;
+
+import org.springframework.stereotype.Component;
+import spring.intro.dto.UserResponceDto;
+import spring.intro.model.User;
+import spring.intro.service.UserService;
+
+@Component
+public class UserMapper {
+    private final UserService userService;
```

Comment:
do you need this?

Reply:
No i don't, i was confused with functionalitty of Mapper class. At first, i tried use it. Later changed way of solution and have forgotten to delete it. Done.